services/bundle_analysis.py

```diff
@@ -81,7 +81,7 @@ class BundleAnalysisMeasurementData(object):
     def __init__(
         self,
         raw_measurements: List[dict],
-        asset_type: BundleAnalysisMeasurementsAssetType,
+        asset_type: Union[BundleAnalysisMeasurementsAssetType, str],
```

Comment:
Yeah I was looking to add an entry to that, but once I started making the changes I realized too many things (a lot of it on the ingestion side) would need to be updated. So I thought this would be the least disruptive way of doing it.